test/FIRParser/basic.fir

```diff
@@ -178,7 +179,7 @@ circuit MyModule :     ; CHECK: firrtl.circuit "MyModule" {
     auto <= add(UInt<10>(42), UInt<8>("hAB"))
 
     ; CHECK: %c-85_si8 = firrtl.constant(-85 : si8) : !firrtl.sint<8>
-    auto <= add(s8, SInt<8>("hAB"))
```

Comment:
`auto` is unsigned, but the result of the expression `add(s8, SInt<8>("hAB"))` is signed, so I've added a new output parameter called `sauto` that is signed, and can be connected to here.